_pages/production.md

```diff
@@ -24,20 +24,20 @@ sidenav:
     href: "#certificate-rotation-process"
 saml_year: 2023
 ---
-<div class="usa-alert usa-alert--warning">
-<div class="usa-alert__body">
-<p class="usa-alert__text"><b> Please note: The deployment process can take up to two weeks to complete. You may wish to have a grace period between when we deploy and when you add the Login.gov button to your site. </b></p>
-</div>
-</div>
+{% capture deployment_warning %}
+Please note: The deployment process can take up to two weeks to complete. You may wish to have a grace period between when we deploy and when you add the Login.gov button to your site.
```

Comment:
This was bolded previously.

Reply:
I think since it is already highlighted as a warning having it bolded is a bit excessive? None of the warnings on other pages are bolded that I have seen